docs/csharp/programming-guide/concepts/linq/linq-to-xml-overview.md

```diff
@@ -5,6 +5,10 @@ ms.assetid: 716b94d3-0091-4de1-8e05-41bc069fa9dd
 ---
 # LINQ to XML Overview (C#)
 
+LINQ to XML provides an in-memory XML programming interface that leverages the .NET Language-Integrated Query (LINQ) Framework. LINQ to XML uses the latest .NET Framework language capabilities and is comparable to an updated, redesigned Document Object Model (DOM) XML programming interface. 
+ 
+ The LINQ family of technologies provides a consistent query experience for objects (LINQ to Objects), relational databases (LINQ to SQL), and XML (LINQ to XML).  
```

Comment:
This doesn't really go here, but it is a good sentence to add before "You can write LINQ queries in C#..." in <https://docs.microsoft.com/dotnet/csharp/programming-guide/concepts/linq/index>